hotel-management/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "counter-app",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
```

Comment:
`--ext js,jsx --report-unused-disable-directives --max-warnings 0` what this mean?

Reply:
here's what I know

- --ext js,jsx: file extensions that ESLint should look at, but I use TypeScript here so I will fix the extensions.